app/views/settings/redmine_git_hosting/_gitolite_recycle_bin.html.slim

```diff
@@ -7,8 +7,9 @@ h3 = l(:label_empty_recycle_bin)
       th = l(:label_recycle_bin_content_size)
       th
         = l(:label_delete_from_recyle_bin)
-        = surround '(', ')' do
```

Comment:
Hi Alexander! Thanks for your work :+1: 
Just for your information this is pure HAML helper : http://haml.info/docs/yardoc/file.REFERENCE.html#surround

Reply:
Ok... I've just seen you switched to slim ^^